README.md

```diff
@@ -115,15 +115,25 @@ python3 mqtt-simulator/main.py -f <path/settings.json>
     | Key | Type | Description | Required |
     | --- | --- | --- | --- |
     | `NAME` | string | JSON property name to be sent | yes |
-    | `TYPE` | string | It can be `"int"`, `"float"` or `"bool"` | yes |
+    | `TYPE` | string | It can be `"int"`, `"float"`, `"bool"` or `"expression"` | yes |
     | `INITIAL_VALUE` | number or bool (same as defined in `TYPE`) | Initial value that the property will assume when the simulation starts (random otherwise) | optional |
-    | `MIN_VALUE` | number | Minimum value that the property can assume | if `TYPE` is different from `"bool"` |
-    | `MAX_VALUE` | number | Maximum value that the property can assume | if `TYPE` is different from `"bool"` |
-    | `MAX_STEP` | number | Maximum change that can be applied to the property from a published data to the next | if `TYPE` is different from `"bool"` |
+    | `MIN_VALUE` | number | Minimum value that the property can assume | if `TYPE` is different from `"bool"` or `"expression"` |
```

Comment:
Since we now have more types, I think it's better to mention the types for which this field is mandatory rather than mentioning the exceptions.
The same goes for the other properties that are described like this.

```suggestion
    | `MIN_VALUE` | number | Minimum value that the property can assume | if `TYPE` is `"int"` or `"float"` |
```